risc0/build/Cargo.toml

```diff
@@ -19,6 +19,8 @@ risc0-zkvm-platform = { workspace = true }
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_json = "1.0"
 tempfile = "3"
+which = "6.0.1"
+backoff = "0.4.0"
```

Comment:
```suggestion
which = "6.0"
backoff = "0.4"
```

Reply:
Please sort deps

Reply:
Dependencies sorted now.

And given the feedback about checking if docker is running, I ended up deleting these dependencies as I removed the entire docker check.